test/extended/networking/util.go

```diff
@@ -696,3 +697,95 @@ func GetIPAddressFamily(oc *exutil.CLI) (bool, bool, error) {
 	}
 	return hasIPv4, hasIPv6, nil
 }
+
+func createSleepingPod(f *e2e.Framework, nodeName string, generateName string, labels map[string]string) (*v1.Pod, error) {
+	contName := fmt.Sprintf("%s-container", generateName)
+	pod := &v1.Pod{
+		TypeMeta: metav1.TypeMeta{
+			Kind: "Pod",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			GenerateName: generateName,
+			Labels:       labels,
+		},
+		Spec: v1.PodSpec{
+			Containers: []v1.Container{
+				{
+					Name:    contName,
+					Image:   imageutils.GetE2EImage(imageutils.Agnhost),
+					Command: []string{"sleep", "100000"},
+				},
+			},
+			NodeName:      nodeName,
+			RestartPolicy: v1.RestartPolicyNever,
+		},
+	}
+
+	p, err := f.ClientSet.CoreV1().Pods(f.Namespace.Name).Create(context.Background(), pod, metav1.CreateOptions{})
+	if err != nil {
+		return nil, err
+	}
+
+	err = frameworkpod.WaitForPodRunningInNamespace(f.ClientSet, p)
+	return p, err
+}
```

Comment:
quick question Ori: Can `createPod` in this same file be used instead of `createSleepingPod` or merged together? They seem fairly similar.

Reply:
done - changed the createPod to accept tweaks
removed the other function